src/main/java/frc/robot/subsystems/climber/Climber.java

```diff
@@ -0,0 +1,150 @@
+package frc.robot.subsystems.climber;
+
+import static frc.robot.subsystems.climber.ClimberConstants.*;
+
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.lib.team3015.subsystem.FaultReporter;
+import frc.lib.team6328.util.TunableNumber;
+import frc.robot.subsystems.subsystem.SubsystemIOInputsAutoLogged;
+
+import org.littletonrobotics.junction.Logger;
+
+/**
+ * Models a generic subsystem for a rotational mechanism. The other subsystems defined in this
+ * library aren't good examples for typical robot subsystems. This class can serve as an example or
+ * be used for quick prototyping.
+ */
+public class Climber extends SubsystemBase {
+
+  // these Tunables are convenient when testing as they provide direct control of the subsystem's
+  // motor
+  private final TunableNumber leftMotorPower = new TunableNumber("Climber/leftPower", 0.0);
```

Comment:
I'm not sure if we need any tunables with Tuner X. We can use that for tuning instead.